src/auth/ha-password-manager-polyfill.ts

```diff
@@ -0,0 +1,82 @@
+import { DataEntryFlowStep } from "../data/data_entry_flow";
+
+const ENABLED_HANDLERS = [
+  "homeassistant",
+  "legacy_api_password",
+  "command_line",
+];
+
+export class PasswordManagerPolyfill {
```

Comment:
We could make this a LitElement without a shadowroot? Add a `createRenderRoot` function that returns `this`.